avm/utilities/pipelines/staticValidation/psrule/ps-rule.yaml

```diff
@@ -70,6 +70,7 @@ configuration:
       "ssoClientSecretKeyVaultPath",
       "ssoSecretType",
       "tokenValidityLength",
+      "uniqueKeyPolicyKeys"
```

Comment:
Awesome job again @seesharprun and great contribution. Since this update would trigger all pipelines, if you agree we'd like to have that modified before approving and merging this PR. Usually we test all modules whenever a change impacts shared files like this one, but in this case we think it would be an overkill.
Please bare with us managing that prior to approving this one 🙏

Reply:
I approve!

Thanks for the thorough feedback.

Once this is merged in, I can start on some contributions to add functionality. The goal here was to get to a working baseline.

Thanks again!

Reply:
Keeping this open until the psrule fix gets merged

Reply:
Done via #3587. @seesharprun please pull latest from upstream, you should have no changes needed on psrule.yml.
Please close this comment once done

Reply:
I just pulled latest